docs/REST API/rest-api.md

```diff
@@ -9582,7 +9625,7 @@ _Available in Fleet Premium._
 
 Update a package to install on macOS, Windows, or Linux (Ubuntu) hosts.
 
-`PATCH /api/v1/fleet/software/titles/:title_id/package`
+`PATCH /api/v1/fleet/software/titles/:id/package`
```

Comment:
Just noticed this parameter isn't documented in the table below; mind making a separate PR to fix this + document it in the "Parameters" table below? (You can leave this change as-is since it'll get merged in later.

I can't suggest a change to the parameters table in my review, but should be at the top of the list and look like:
```
| id | integer | path | ID of the software title being updated. |
```

Reply:
@rachaelshaw Thanks for catching this! Updated in a new PR:

https://github.com/fleetdm/fleet/pull/24643